app/topics/shared/topic.model.ts

```diff
@@ -21,24 +21,44 @@ export class Topic {
   subTopics: Topic[];
   supervisors: User[];
   title: string;
-
+  studentID: number[];
+  supervisorID: number[];
+  reviewerID: number[];
```

Comment:
I think we don't need extra variables for userIds, since everyone can get the id, by `user.id`. Furthermore having an extra list for ids, can lead to inconsistency.